.github/workflows/lint.yml

```diff
@@ -5,14 +5,13 @@ on: [pull_request]
 jobs:
   assert-formatted:
     name: Assert that code has been linted and formatted before merging
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Install swiftformat and swiftlint 
+      - name: Install binny to run lint tools 
```

Comment:
added a recommendation in the other PR that we can use a script/deno.land/x rather than a binary, it might effect here.